accounts/keystore/keystore.go

```diff
@@ -476,20 +476,8 @@ func (ks *KeyStore) ImportSingleExtendedKey(extKey *extkeys.ExtendedKey, passphr
 		ExtendedKey: extKey,
 	}
 
-	// if account is already imported, return cached version
 	if ks.cache.hasAddress(key.Address) {
-		a := accounts.Account{
-			Address: key.Address,
-		}
-		ks.cache.maybeReload()
-		ks.cache.mu.Lock()
-		a, err := ks.cache.find(a)
-		ks.cache.mu.Unlock()
-		if err != nil {
-			zeroKey(key.PrivateKey)
-			return a, err
-		}
-		return a, nil
+		return accounts.Account{}, fmt.Errorf("account already exists")
```

Comment:
```suggestion
		return accounts.Account{}, ErrAccountExists
```

I think it's likely we will need to handle this error, hence I would make a variable out of it.

Reply:
@adambabik it makes sense, I added it with the second commit.